frontend/test/auth/auth.store.spec.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { createStore, combineReducers } from 'redux';
+import { AuthActions } from '@/v5/store/auth/auth.redux';
+import { authMockFactory } from './auth.fixtures';
+import { selectAuthenticationFetched, selectIsAuthenticated, selectIsPending, selectLoginError, selectReturnUrl } from '@/v5/store/auth/auth.selectors';
+import reducers from '@/v5/store/reducers';
+
+describe('Auth: store', () => {
+	let dispatch, getState = null;
+
+	const mockAuth = authMockFactory();
+
+	beforeEach(() => {
```

Comment:
I'm wondering if we can move this to the test.helpers.ts file

Reply:
Yes, that would be a good idea,

Reply:
If you do so, can you please modify the xisting test to use this?